huaweicloud/config/endpoints.go

```diff
@@ -423,6 +423,10 @@ var allServiceCatalog = map[string]ServiceCatalog{
 		Name:    "dms",
 		Version: "v2",
 	},
+	"servicestagev2": {
```

Comment:
just one endpoint , maybe can remove version

Reply:
There will also be a v1 endpoint in the future, I want to use it as default.